geometry_msgs/PointStamped pt
uint16 camera_id
uint8 point_id